demo-shell/src/app/components/files/files.component.ts

```diff
@@ -112,7 +134,8 @@ export class FilesComponent implements OnInit, OnChanges, OnDestroy {
                 private translateService: TranslationService,
                 private router: Router,
                 @Optional() private route: ActivatedRoute,
```

Comment:
Would you be so kind as to put it at the end of the parameter list, being as an optional parameter. I'd really appreciate it.